lib/balrog/routes_middleware.rb

```diff
@@ -0,0 +1,30 @@
+require 'bcrypt'
+require_relative 'helpers'
+
+# Public: Balrog middleware that handles form submissions, checking the
+# password against the configured hash, and setting a session variable if
+# they match.
+# 
+# In order to protect SideKiq Web you would do something like this: 
+#
+# require 'sidekiq/web'
+#
+# Sidekiq::Web.disable(:sessions)
+# Sidekiq::Web.use Balrog::RoutesMiddleware
+#
+# mount Sidekiq::Web => '/sidekiq'
+
+class Balrog::RoutesMiddleware < Balrog::Middleware
```

Comment:
Do you need to inherit from `Balrog::Middleware` at all? It doesn't really look like you're reusing anything from that middleware. All this middleware does is render the gate if you're not authenticated.